### PR TITLE
feat: add 2 weight functions in multi-label

### PR DESCRIPTION
**Pull request by thsno02**
根据 https://github.com/PaddlePaddle/PaddleClas/issues/2951 新增两个 ratio2weight 函数。代码里延续论文中的命名，所以原 `ratios2weight` 改为 `ratios2weight_1`。

Type 3 示例：

```
# loss function config for traing/eval process
Loss:
  Train:
    - MultiLabelLoss:
        weight: 1.0
        weight_type: 3 
        weight_alpha: 1 # float
        weight_ratio: True
        size_sum: True
  Eval:
    - MultiLabelLoss:
        weight: 1.0
        weight_type: 3 
        weight_alpha: 1 # float
        weight_ratio: True
        size_sum: True
```

Type 2/1 示例：

```
# loss function config for traing/eval process
Loss:
  Train:
    - MultiLabelLoss:
        weight: 1.0
        weight_type: 2 
        weight_ratio: True
        size_sum: True
  Eval:
    - MultiLabelLoss:
        weight: 1.0
        weight_type: 2
        weight_ratio: True
        size_sum: True
```

但是 https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/ppcls/loss/dmlloss.py 中 import 了 `ratio2weight` 这个函数，目前的解决方案是在 dmlloss 中开启 weight_type 和 weight_alpha 这两个参数，与 multilabelloss 中保持一致，然后计算对应的 loss。

### CODE REVIEW

File: ppcls/loss/multilabelloss.py [hunk 1]
```diff
@@ -3,7 +3,14 @@
 import paddle.nn.functional as F
 
 
-def ratio2weight(targets, ratio):
+def ratio2weight_1(targets, ratio):
```
Comment (cuicheng01) [inline ppcls/loss/multilabelloss.py:6]: 这里的1、2、3是官方repo的命名方式吗？

Comment (thsno02) [reply ppcls/loss/multilabelloss.py:6]: https://github.com/valencebond/Rethinking_of_PAR/blob/master/tools/function.py 官方 repo 只有一个，命名中无后缀。1、2、3 是按论文给的顺序给出来的。

Comment (cuicheng01) [reply ppcls/loss/multilabelloss.py:6]: 建议这里只定义一个函数ratio2weight，里边增加参数mode，mode可以是1，2，3，在注释中说明下区别
